### PR TITLE
Add butterflow

### PR DESCRIPTION
**Pull request by TheRandomLabs**
I had to use a slightly convoluted checkver because the release tags aren't equal to the version strings used in the file names.

**Copied from README:**

BF is a video manipulation tool that can:

1. Make **motion interpolated videos** (increase a video's frame rate by rendering intermediate frames based on motion using a combination of pixel-warping and blending).
2. Make **smooth motion videos** (do simple blending between frames).
3. Leverage interpolated frames to make **fluid slow motion videos**.

### CODE REVIEW

**Comment by linsui** [comment]
```
Windows 10 (Portable): Download the latest releases.

    Preview: butterflow-0.2.4a4-win64.zip
        Sha256: 856ad05bafaeb9d084d6b1cbf0bdec17defeef438e610f6a3a772087acea301b
    Stable: butterflow-0.2.3-win64.zip
        Sha256: a77fbbdbdd0d85bb31feac30e53378a36ff4fb2a8a98ba15a6fa06def4b36ad1
```

Please extract hash from readme. Why not use stable version?

**Comment by TheRandomLabs** [comment]
Would it be worth having a separate manifest for the alpha version?

File: bucket/butterflow.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
         "architecture": {
             "64bit": {
                 "url": "https://github.com/dthpham/butterflow/releases/download/$matchRel/butterflow-$version-win64.zip",
+                "hash": {
+                    "url": "https://raw.githubusercontent.com/dthpham/butterflow/master/README.md",
+                    "find": "$basename[\\S\\s]+\\* Sha256: $sha256"
```
Comment (Ash258) [inline bucket/butterflow.json]: Multilne regex is needed
```suggestion
                    "regex": "(?sm)$basename.*?\\*\\s+Sha256:\\s+$sha256"
```

File: bucket/butterflow.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "architecture": {
         "64bit": {
             "url": "https://github.com/dthpham/butterflow/releases/download/0.2.4.alpha2/butterflow-0.2.4a4-win64.zip",
-            "hash": "856ad05bafaeb9d084d6b1cbf0bdec17defeef438e610f6a3a772087acea301b",
+            "hash": "a77fbbdbdd0d85bb31feac30e53378a36ff4fb2a8a98ba15a6fa06def4b36ad1",
```
Comment (Ash258) [inline bucket/butterflow.json]: Wrong hash

File: bucket/butterflow.json [hunk 3]
```diff
@@ -20,6 +20,10 @@
         "architecture": {
             "64bit": {
                 "url": "https://github.com/dthpham/butterflow/releases/download/$matchRel/butterflow-$version-win64.zip",
+                "hash": {
```
Comment (Ash258) [inline bucket/butterflow.json]: Move hash out of architecture property.

**Comment by TheRandomLabs** [comment]
@linsui Thanks for the suggestion. I hope I've done it correctly. :) Also, the stable version is over a year older than the preview version, and [butterflow-ui](https://github.com/wagesj45/butterflow-ui) seems to deem it stable enough to use it in their release build. However, I can change it to the stable version if you would like.

File: bucket/butterflow.json (hunk 1 above)
Comment (linsui) [reply bucket/butterflow.json]: `$basename\\s+\\* Sha256: $sha256` is enough.

File: bucket/butterflow.json (hunk 2 above)
Comment (TheRandomLabs) [reply bucket/butterflow.json]: I got the right hash for 0.2.4.alpha2. `a77fbbdbdd0d85bb31feac30e53378a36ff4fb2a8a98ba15a6fa06def4b36ad1` is for 0.2.3.

File: bucket/butterflow.json (hunk 1 above)
Comment (TheRandomLabs) [reply bucket/butterflow.json]: Thanks, I don't have much experience with regular expressions.

**Comment by linsui** [comment]
@TheRandomLabs If it's stable enough, it's Ok 😄

**Comment by TheRandomLabs** [comment]
@Ash258 I've applied your suggestions. :)

**Comment by TheRandomLabs** [comment]
Regarding the description, what was wrong with `a command-line tool for performing motion interpolation on videos`?

**Comment by Ash258** [comment]
- `Command line tool` is obsolete for Main bucket
- Motion interpolation is not the only one feature this application can do

**Comment by TheRandomLabs** [comment]
How about `a tool for making motion interpolated and smooth motion videos`?

**Comment by TheRandomLabs** [comment]
Fair enough.

**Pull request merged**